openfisca_uk/parameters/consumption/carbon/production/carbon(goods &services).yaml

```diff
@@ -0,0 +1,13 @@
+description: Carbon Emissions, Production of goods and emissions
```

Comment:
```suggestion
description: Carbon emissions produced in the UK.
```

Reply:
As  it deals with  production of goods and services, I have provided the description in this manner.

Reply:
How about `Carbon emissions from the production of goods and services in the UK`? I think we can uncapitalise `Emissions` and `Production` also.

Reply:
Done

Reply:
Thanks, just the uncapitalising to go I think.

Reply:
sure